release-notes/DxMs/DataAPI_change_log.md

```diff
@@ -0,0 +1,67 @@
+---
+uid: DataAPI_change_log
+---
+
+# Data API change log
+
+> [!IMPORTANT]
+> At present, Data API is in soft launch and as such this feature is only available in preview if the [DataAPI](xref:Overview_of_Soft_Launch_Options#dataapi) soft-launch option is enabled.
```

Comment:
Activating the soft launch requires more than just setting the flag. Can we link to xref:Data_Sources_Setup (https://docs.dataminer.services/user-guide/Advanced_Modules/Data_Sources/Data_Sources_Setup.html) instead of linking to the soft-launch option?